src/content/docs/configuration/sched-ext.md

```diff
@@ -70,50 +74,147 @@ sudo systemctl start scx
 sudo systemctl stop scx
 ```
 
-## Brief introduction to the main ones
+### Adding flags to the scheduler
 
-Since there are many schedulers to choose from, we want to give a little introduction about the schedulers in hand:
+You can include any flag to the sched-ext schedulers. Check out this [guide.](<https://github.com/sched-ext/scx/blob/main/services/README.md>)
```

Comment:
We should put this as a link at the bottom of the page under "See More". This way, we can also add other references, e.g. andrea's and changwoo's blog posts.